00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/Content Lists.md

```diff
@@ -0,0 +1,40 @@
+## Sorting of lists, to aid readability
+
+### Decisions and conclusions
+
+- Sort on Alias, not Plugin ID
+- Avoid comment markers
```

Comment:
What do you mean by comment markers?

Reply:
Wording improved in 2ff0699c58904a2c298e6bfce9192916d3b9b43c